tests/test_com_task.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Domrachev Ivan, Simeon Nedelchev
+
+"""Test fixture for the com task."""
+
+import unittest
+
+import numpy as np
+import pinocchio as pin
+from robot_descriptions.loaders.pinocchio import load_robot_description
+
+from pink import Configuration
+from pink.tasks import ComTask, TargetNotSet
+
+
+class TestComTask(unittest.TestCase):
+    """Test consistency of the com task.
+
+    Note:
+        This fixture only tests the task itself. Integration tests with the IK
+        are carried out in :class:`TestSolveIK`.
```

Comment:
OK for only testing the task itself in this file, but there is no integration test yet. Can you add one to `TestSolveIK`?

It can be close to `test_three_tasks_fulfilled`, with the CoM task rather than a pelvis task.

Reply:
Sure, we will ad integration test as well

Reply:
There are two new tests in `TestSolveIK` based on `test_three_tasks_fulfilled`: `test_com_task_fulfilled` and `test_com_task_convergence`